html/modules/custom/reliefweb_entities/src/Entity/Report.php

```diff
@@ -11,13 +11,17 @@
 use Drupal\reliefweb_entities\BundleEntityInterface;
 use Drupal\reliefweb_entities\DocumentInterface;
 use Drupal\reliefweb_entities\DocumentTrait;
+use Drupal\reliefweb_entities\OpportunityDocumentTrait;
```

Comment:
Leftover.

Reply:
It's still using `updateModerationStatusFromSourceStatus`

Reply:
@attiks I would suggest to move `updateModerationStatusFromSourceStatus` to the `DocumentTrait` then. This trait is notably already used by the Job, Training and Report classes.

Reply:
Moved, added test for blocked sources